todo/src/reducers/reducers.js

```diff
@@ -0,0 +1,38 @@
+import {ADD_TODO, TOGGLE_TODO} from '../actions';
+
+const initialState = {
+    todos: [
+        {
+            text:'initialState text',
+            id: Date.now(),
+            completed: false,
+        },
+    ]
+};
+
+export default (state = initialState, action) => {
+    switch (action.type) {
+        case ADD_TODO:
+            const newTodo = {
+                text:action.payload,
+                id: Date.now(),
```

Comment:
remember in the interest of keeping the reducer functions 100% pure, there should be no functions that return different results when invoked with the exact same circumstances (which is what defines a pure function). per the redux docs, this includes random or time-locked methods like Date.now, Math.random, or a third-party library like `uuid` that generates a random id for you. this declaration of a newTodo should be moved to the action creator and passed here as the payload